llvm/test/CodeGen/RISCV/GlobalISel/legalizer/rvv/legalize-xor.mir

```diff
@@ -9,8 +9,8 @@ body:             |
     ; CHECK-LABEL: name: test_nxv1i8
```

Comment:
This seems unrelated to this patch?

Reply:
Not addressed. Should this change be committed as its own NFC?

Reply:
I'll commit it as its own NFC. It keeps making its way in again. I've removed it 2x since you commented here!